homeassistant/components/flexit_bacnet/config_flow.py

```diff
@@ -0,0 +1,78 @@
+"""Config flow for Flexit Nordic (BACnet) integration."""
+from __future__ import annotations
+
+import asyncio.exceptions
+import logging
+from typing import Any
+
+from flexit_bacnet import FlexitBACnet
+from flexit_bacnet.bacnet import DecodingError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_DEVICE_ID, CONF_IP_ADDRESS
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_DEVICE_ID = 2
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_IP_ADDRESS): str,
+        vol.Required(CONF_DEVICE_ID, default=DEFAULT_DEVICE_ID): int,
+    }
+)
+
+
+async def validate_input(data: dict[str, Any]) -> str:
```

Comment:
This function is fairly small, I wouldn't mind if you inlined this

Reply:
Fixed